runtime/zeitgeist/src/parameters.rs

```diff
@@ -109,6 +109,8 @@ parameter_types! {
 
     // Balance
     pub const ExistentialDeposit: u128 = 5 * MILLI;
+    pub const MaxHolds: u32 = 1;
+    pub const MaxFreezes: u32 = 1;
```

Comment:
Seems fine to me. I don't think we use freezes or holds yet, so... If you have a suggestion though, let's hear it.

Reply:
I just thought about setting the values to zero instead, because it was like this before. But maybe Harald had a particular reason for using one for both.